raiden/tests/utils/transport.py

```diff
@@ -160,11 +157,7 @@ def generate_config(port: int):
             )
         return server_name, config_file
 
-    try:
-        yield generate_config
-    finally:
-        if delete_base_dir:
-            shutil.rmtree(synapse_base_dir)
```

Comment:
- How is this related to this PR?
- Don't we need to clean up the `synapse_base_dir` any more?

Reply:
Synapse failed to run because of a import cycle. Having the file was useful to actually find the problem, since the error message was being redirected to devnull. I can remove it if you want.

Reply:
No strong feelings on my side. If you think we should remove it that's fine.